packages/@lwc/ssr-compiler/src/compile-js/generate-markup.ts

```diff
@@ -87,10 +89,17 @@ const bAssignGenerateMarkupToComponentClass = esTemplate`
 export function addGenerateMarkupExport(
     program: Program,
     state: ComponentMetaState,
+    options: TransformOptions,
```

Comment:
Not sure what @nolanlawson thinks about this, but it might be worth scoping this down to `{ name: string, namespace: string }`.

Reply:
[Updated](https://github.com/salesforce/lwc/pull/4847/files#diff-0300e0d1ad482c59fb69f8d1673e59d3355f5653b5b95a2c9527d7623352818dR26-R27)!